### PR TITLE
Add peers

### PR DESCRIPTION
**Pull request by Chris-Sigopt**
Add some peer dependencies to yarn.

### CODE REVIEW

**Comment by tjs-intel** [comment]
Are we using these modules directly? I don't understand why amazon doesn't require them directly

File: ci/check_node_deps.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 set -e
 set -o pipefail
 
-DEPCHECK_IGNORES="@babel/register,@fortawesome/fontawesome-free,buffer,lodash-es,minimist,process,webpack-cli"
+DEPCHECK_IGNORES="@babel/register,@fortawesome/fontawesome-free,buffer,lodash-es,minimist,process,webpack-cli,@aws-sdk/client-sso-oidc,@aws-sdk/client-sts"
```
Comment (tjs-intel) [inline ci/check_node_deps.sh:8]: mind sorting these before merging?

**Comment by Chris-Sigopt** [comment]
I am very confused by the AWS deps as well: based off of this: https://github.com/npm/rfcs/blob/main/implemented/0025-install-peer-deps.md it looks like npm 4 years ago switched to automatically install any missing peer deps like these are. The fact that the CI started failing so recently, on what looks like an unrelated update, also feels off. I feel like there is a better way to fix this, just getting it to work so I can get sigopt-python working!

**Review by tjs-intel — APPROVED** [review]
Thanks for getting it working again!

**Pull request merged**